plugin/environment.world

```diff
@@ -59,25 +65,45 @@
         <uri>model://teensize_goal</uri>
       </include>
       <pose>4.525 0 0 0 0 0</pose>
-    </model> 
+    </model>
 
     <!-- The ball -->
     <model name="nubots_ball">
       <include>
         <uri>model://teensize_ball</uri>
       </include>
       <pose>0 0 0.15 0 0 0</pose>
-      <plugin name="nubots_ball" filename="libnubotsball_plugin.so"/>
-    </model> 
+      <plugin name="nubots_ball" filename="libnubots_ball_plugin.so">
+        <update_rate>100</update_rate>
+      </plugin>
+    </model>
 
-    <!-- The NUbots' Igus -->
-    <model name="nubots_igus">
+    <!-- The NUbots' NUgus -->
+    <model name="nubots_nugus">
       <include>
-        <uri>model://nubots_igus_hull</uri>
+        <uri>model://nubots_nugus_hull</uri>
       </include>
       <pose>-0.15 0.125 0.6225 0 0.125 0</pose>
       <!-- Attach the plugin to this model -->
-      <plugin name="nubots_igus" filename="libnubotsigus_plugin.so"/>
+      <plugin name="nubots_nugus" filename="libnubots_nugus_plugin.so">
+        <update_rate>100</update_rate>
+        <!-- Joint spring stiffness coefficient -->
+        <joint_stiffness>3.25</joint_stiffness>
```

Comment:
This value is actually in the model itself in https://github.com/NUbots/Gazebo/blob/houliston/refactor/models/nubots_nugus_hull/model.sdf#L170 
Maybe we should just do this over there instead as changing them dynamically vs on the model seems strange. Maybe more of these values go on the model too.